Editor/PluginSettings/FirstTimeSettingsLauncher.cs

```diff
@@ -1,38 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
-using UnityEditor;
-using UnityEngine;
-
-namespace AmazonGameLift.Editor
-{
-    /// <summary>
-    /// Shows the settings window once; started by Unity.
-    /// </summary>
-    internal sealed class FirstTimeSettingsLauncher
-    {
-        private const string NonEmptyValue = "1";
-
-        [InitializeOnLoadMethod]
-        private static void Run()
-        {
-            if (Application.isBatchMode)
-            {
-                return;
-            }
-
-            GetSettingResponse response = CoreApi.SharedInstance.GetSetting(SettingsKeys.WasSettingsWindowShown);
```

Comment:
If any of these setting keys are no longer used with this PR, we should be removing them